src/pool.rs

```diff
@@ -34,6 +34,15 @@ impl<T: TaskCell + Send> ThreadPool<T> {
         self.remote.spawn(t);
     }
 
+    /// Scale workers of the thread pool, the new thread count should be
+    /// smaller than the max_thread_count, otherwise it will become no-op
```

Comment:
Perhaps should be adjusted to `max_thread_count` to keep consistent with `Builder`.